Cargo.lock

```diff
@@ -365,12 +365,14 @@ checksum = "cf1de2fe8c75bc145a2f577add951f8134889b4795d47466a54a5c846d691693"
 
 [[package]]
 name = "bitvec"
```

Comment:
a stdlib type of crate, makes me think that we should probably create a list of crates which are OK for upgrades without reviews.